pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.amazonaws</groupId>
   <artifactId>amazon-sqs-java-messaging-lib</artifactId>
-  <version>1.0.8</version>
+  <version>1.0.8-DH-RPS</version>
```

Comment:
rps? :D

Reply:
I was also thinking whether to use `<classifier>` for the patch suffix :thinking: